File: 2298-count-integers-with-even-digit-sum/2298-count-integers-with-even-digit-sum.js
/**
 * @param {number} num
 * @return {number}
 */
var countEven = function(num) {
     f=0
    for(i=1;i<=num;i++){
        n=String(i).split("")
        r=n.reduce((acc,e)=> acc+Number(e),0)
        if(r%2==0){
            f++
        }
      
    }
    return f
};

File: 2298-count-integers-with-even-digit-sum/2298-count-integers-with-even-digit-sum.go
func countEven(num int) int {
    ans:=0
    for i := 2; i <= num; i++ {
		sum := 0
		for x := i; x > 0; x /= 10 {
			sum += x % 10
		}
		if sum%2 == 0 {
			ans++
		}
	}
	return ans
}